### PR TITLE
Add missing rfc8654 capability parsing

### PR DESCRIPTION
**Pull request by ton31337**

### CODE REVIEW

**Comment by ton31337** [comment]
@infrastation do you mind reviewing this, please?

**Comment by ton31337** [comment]
Anyone from maintainers can look at these trivial changes?

File: tests/bgp-extended-msg.out [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	      32-Bit AS Number (65), length: 4
+		 4 Byte AS 65001
+	    Option Capabilities Advertisement (2), length: 2
+	      BGP Extended Message (6), length: 0
+	    Option Capabilities Advertisement (2), length: 6
+	      Multiple Paths (69), length: 4
+		AFI IPv4 (1), SAFI Unicast (1), Send/Receive: Receive
+	    Option Capabilities Advertisement (2), length: 18
+	      Unknown (73), length: 16
+		no decoder for Capability 73
```
Comment (fxlb) [inline tests/bgp-extended-msg.out:26]: Why this Capability 73 (unkown) is present?
Seems to be linked to an expired draft?

Comment (ton31337) [reply tests/bgp-extended-msg.out:26]: It's because https://github.com/the-tcpdump-group/tcpdump/pull/939 was rejected :)

**Comment by fxlb** [comment]
OK, I remember now.

**Comment by fxlb** [comment]
Merged, than you!

**Comment by ton31337** [comment]
@fxlb thank you, would you mind reviewing https://github.com/the-tcpdump-group/tcpdump/pull/953 and https://github.com/the-tcpdump-group/tcpdump/pull/962 as well?

**Pull request merged**